src/components/Menus/Menus.tsx

```diff
@@ -29,15 +29,14 @@ const MenusContainer = styled.div`
 const MenuHeading = styled.div<{ isOpen: boolean }>`
   padding: 4px 8px;
   cursor: pointer;
-  border: 1px solid transparent;
-  border-bottom: none;
 `
 
 const MenuContainer = styled.div<{ isEnabled: boolean }>`
   position: relative;
 
   & ${MenuHeading} {
-    background-color: #fff;
+    background-color: #fafafa;
+    line-height: 24px;
```

Comment:
Probably would be a better practice to set line height in em units.

Reply:
Probably yes, but everything around is defined in pixels. It does not play any role in this case to use em units.